vllm/v1/sample/ops/topk_topp_sampler.py

```diff
@@ -264,7 +264,7 @@ def flashinfer_sample(
     Statistically, this function is equivalent to the `random_sample` function.
     However, this function is faster because it avoids sorting the logits tensor
     via rejection sampling.
-    
+
```

Comment:
Can you update the function comments to match the new behavior?

Reply:
Good point, done in 561d1d42644cfc047408f700bc2331d355ac48e3